content/about/index.md

```diff
@@ -10,4 +10,33 @@ Packit provides tooling and automation to integrate upstream open source project
 
 https://github.com/packit-service/packit
 
+
+### The key principles of packit
```

Comment:
How these relate to principles in this page: https://github.com/packit-service/packit/blob/master/docs/source-git.md#principles Are the ones only about source git, or is there some intersection?

Reply:
Whoops, I forgot about this: we should merge them.